### ISSUE TITLE
Bug : table & background color

### ISSUE THREAD

**Issue by s-renier-taonix-fr** [xdan/jodit#722]
**Jodit Version:**  3.6.18 (Playground)

**Browser:**  FF 91.0.2 (64 bits)
**OS:**  Linux Ubuntu 20.04
**Is React App:**  False

**Code**

```
Jodit playground :
- Delete all the content
- Create a table
- Click on a cell
- In the context-menu, click on "background" and choose a color
```
**Expected behavior:**
The color picker must disappears as soon as we choose a color.

**Actual behavior:**
The color picker doesn't disappears when we click on a color
but it's disappears if we click on a second one.

**Comment by xdan** [comment]
Fixed in 3.8.2.

**Comment by s-renier-taonix-fr** [comment]
Thank you xdan.
I close this issue.
